parser_library/test/debugging/debugger_test.cpp

```diff
@@ -239,13 +240,13 @@ bool check_step(
         if (sc.size() != 3U)
             return false;
         using variables = std::vector<hlasm_plugin::parser_library::variable>;
-        variables globs(d.variables(sc.item(0).variable_reference));
+        variables globs(std::move(d.variables(sc.item(0).variable_reference)));
```

Comment:
these moves do not make sense to me... `variables(...)` already returns an r-value, doesn't it?

Reply:
You're right